io.github.qnapi.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.qnapi
+default-branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "5.10"
+command: qnapi
+rename-icon: qnapi
+rename-desktop-file: qnapi.desktop
+copy-icon: true
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --env=TMPDIR=/var/tmp # QtSingleApplication
+  - --filesystem=host
```

Comment:
Does it really need the whole host  filesystem access?

Reply:
I don't see any easy workaround of that without edit of an upstream app because opening files is not a big problem but saving subtitles with the same name as movies it not easy with portals.

Reply:
Maybe limiting to download dir is an option but without an upstream patch error message may be misleading.

Reply:
Well, having home access might be okay in such case. But host is just too wide for a subtitles editing apps. Flatpaks allows you to open a wider filesystem hole easily, either by using CLI or a GUI like https://flathub.org/apps/details/com.github.tchx84.Flatseal